subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/configurations/DefaultConfiguration.java

```diff
@@ -1880,10 +1884,16 @@ public boolean isCanBeConsumed() {
 
     @Override
     public void setCanBeConsumed(boolean allowed) {
+        setCanBeConsumed(allowed, true);
+    }
+
+    /* package */ void setCanBeConsumed(boolean allowed, boolean warn) {
```

Comment:
Perhaps a bigger conversation about standards to be had at some point.  

I don't see this convention as being condescending of others' ability to understand, but rather letting them know that I intended to do this and that I didn't just leave off an access modifier by mistake (and had everything just happen to work).  This is an affirmation of my intent so that readers know they can trust the author, not because the author doesn't trust the readers.

I've done this elsewhere and no one has mentioned it.  Consistency is valuable though.  I feel this is on par with not just things like "use final appropriately" (despite we and the IDE being able to tell if a local changes or not) but with things like "always write javadoc on top level types" as a thinking aide for the author.  In the seconds to type this out you get to consider if this really needs to be package-accessible.

I'd prefer to leave it and continue doing it, unless everyone really hates it.